odoo/addons/base/models/ir_property.py

```diff
@@ -235,6 +257,8 @@ def clean(value):
                     'type': self.env[model]._fields[name].type,
                 })
         self.create(vals_list)
+        if False in values:
+            self.clear_caches()
```

Comment:
Don't we have missing invalidation cases for the new cache, such as direct calls to create() or write() with `res_id == False`?